docs/UserGuide.md

```diff
@@ -670,6 +670,8 @@ Examples:
 **Q**: I would like to contribute to this project. May I know who do I reach out to?  
 **A**: You may reach our PR Department email [here](https://www.youtube.com/watch?v=dQw4w9WgXcQ).
 
+**Q**: Is my data backed up to the internet?  
+**A**: No, your data in FitEgo are saved in the hard disk. You will have to transfer the file, which contains your data, to whichever device you wish to continue using FitEgo on.
```

Comment:
Perhaps it is better to say "saved locally in your computer."

On a side note, i think we have this somewhere on top, near quick start section.

Reply:
oo so should I just linked it there or actually is this question redundant?